pageComponents/IndexPage/IndexPage.tsx

```diff
@@ -20,6 +21,11 @@ export default function IndexPage({ children, ...props }: Props) {
         <Eyecatch />
       </Content>
 
+      <Content>
+        <Headline>Recent Posts</Headline>
+        <RecentPosts />
```

Comment:
I think it would be better if `RecentPosts` contains `Headline`. Because `RecentPosts` sounds a section, not just a list.